chatwoot/chatwoot
# fix: Handle invalid country code in contact details

Fixes #4077 

#### How Has This Been Tested?

- Give invalid value for `country_code`.

Ex:
```
window.addEventListener('chatwoot:ready', function() {
  window.$chatwoot.setUser('<%= user_id %>', {
    email: 'jane@example.com',
    country_code:'YH' 
  });
})
```
- Load the widget

- Start a conversation

- Got to the conversation in dashboard, Check contact details are breaking or not
